app/models/reference.rb

```diff
@@ -1,6 +1,9 @@
 class Reference < StudyRelationship
   self.table_name='study_references'
 
+  has_many :retractions
+  accepts nested attributes for :retractions
```

Comment:
This looks like it would crash when running because the method is `accepts_nested_attributes_for`

Reply:
Fixed